### DISCUSSION TITLE
[1 febbraio 2022 - B] Esercizio 1

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#211] [Esami]
**Esercizio 1 (7 punti):**

Dare la rappresentazione tabellare di un automa che riceve in input una sequenza di bit e dà in output 1 ogni volta che gli ultimi 3 bit, intesi come un intero in Ca2, rappresentano un numero che sia o negativo oppure multiplo di 4, anche con sovrapposizioni; quando non sono stati ricevuti ancora 3 bit, si restituisca 0. Si modifichi poi l’automa in modo da non ammettere sovrapposizioni e si sintetizzi il circuito associato (fino alle espressioni booleane minime), usando un FF di tipo SR per il bit di stato più significativo e FF di tipo T per i restanti bit. Realizzare la parte combinatoria utilizzando una PLA.

**Comment by Elia-Belli** [comment]
![DIO](https://user-images.githubusercontent.com/107944476/212311868-cfc33c40-d6fe-4edb-88f5-4acc845b8322.jpg)

**Comment by fraxb04** [comment]
@Elia-Belli
corretto

**Comment by RaulComaz** [comment]
una domanda ma perchè nella tabella ha scritto prima $S01$ poi $S00$?

**Comment by Elia-Belli** [comment]
> una domanda ma perchè nella tabella ha scritto prima $S01$ poi $S00$?

Nessun motivo, li ho scritti nell'ordine di codifica
